sqlglot/dialects/bigquery.py

```diff
@@ -560,6 +560,9 @@ class Generator(generator.Generator):
             exp.DatetimeAdd: date_add_interval_sql("DATETIME", "ADD"),
             exp.DatetimeSub: date_add_interval_sql("DATETIME", "SUB"),
             exp.DateTrunc: lambda self, e: self.func("DATE_TRUNC", e.this, e.text("unit")),
+            exp.FromTimeZone: lambda self, e: self.func(
+                "DATETIME", self.func("TIMESTAMP", e.this, e.args["zone"]), "'UTC'"
```

Comment:
args.get("zone")

Reply:
it's a required arg right?

Reply:
yes, but if you do error level ignore it could be empty